extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -122,7 +129,9 @@ class AmpYoutube extends AMP.BaseElement {
       this.buildImagePlaceholder_();
     }
 
-    installVideoManagerForDoc(this.getAmpDoc());
+    const ampdoc = this.getAmpDoc();
+    installVideoManagerForDoc(ampdoc);
+    videoManagerForDoc(this.win.document).register(this);
```

Comment:
I'd just recommend using `installVideoManagerForDoc(this.element)`. And, yes, this should definitely not be `win.document`. Is something not working? Maybe this is just a incorrect rebase?